docs/include/attributes-ls.asciidoc

```diff
@@ -8,3 +8,6 @@ with a corresponding change to the VPR settings in
 logstash-docs/docs/versioned-plugins/include/attributes-ls-vpr.asciidoc
 /////
 
+:lsyml: Available in`logstash.yml`.
+:plyml: Available in`pipelines.yml`.
+:bothyml: Available in `logstash.yml` and `pipelines.yml`.
```

Comment:
Planning ahead: I'll use this attributes to indicate which settings are in which files.